src/System.Threading.Tasks.Extensions/src/System/Runtime/CompilerServices/ValueTaskAwaiter.cs

```diff
@@ -0,0 +1,222 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Sources;
+
+#if !NETSTANDARD
+using Internal.Runtime.CompilerServices;
```

Comment:
This namespace does not exist. I am trying to understand which APIs used to live under it, and how to consume them here from now on.

Reply:
Turns out all the code that is supposed to be excluded from NETSTANDARD does not show up in ILSpy in any of the DLLs. So I deleted all those cases.